db/database.go

```diff
@@ -834,12 +841,14 @@ func (context *DatabaseContext) Authenticator(ctx context.Context) *auth.Authent
 }
 
 // Makes a Database object given its name and bucket.
-func GetDatabase(context *DatabaseContext, user auth.User) (*Database, error) {
-	return &Database{DatabaseContext: context, user: user}, nil
+func GetDatabase(ctx context.Context, dbCtx *DatabaseContext, user auth.User) (*Database, error) {
+	ctx = dbCtx.AddDatabaseLogContext(ctx) // add db log info before passing ctx down
+	return &Database{Ctx: ctx, DatabaseContext: dbCtx, user: user}, nil
```

Comment:
Do you think we'll be removing Database.Ctx in a future PR, and instead have Database functions accept a context?  Or is this a case where it makes sense to store a context on the struct.

Reply:
hmm... don't see this comment when browsing changed files (deleted?)... anyway, think we're using that Database struct to handle blip sync requests (as we use handler for API), so makes sense to me keep it... seems to be a bigger change (than logging) with blip sync setup if we remove that ctx.